Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2446,37 +2446,46 @@ def get_remote_data_command(client: Client, args: Dict[str, Any], params: Dict)
                 'EntryContext': comments_context
             })
 
-    if ticket.get('closed_at'):
-        if params.get('close_incident'):
-            demisto.debug(f'ticket is closed: {ticket}')
-            entries.append({
-                'Type': EntryType.NOTE,
-                'Contents': {
-                    'dbotIncidentClose': True,
-                    'closeNotes': f'From ServiceNow: {ticket.get("close_notes")}',
-                    'closeReason': converts_state_close_reason(ticket.get("state"))
-                },
-                'ContentsFormat': EntryFormat.JSON
-            })
+    # Handle closing ticket/incident in XSOAR as chosen by close_incident_multiple_options parameter
+    if ticket.get('closed_at') and params.get('close_incident_multiple_options') == 'closed':
```

Comment:
You can merge this two if's together with an or between them, and then continue the function as is, no need to split it and create a new function for it

Reply:
Done.